nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-cluster/src/main/java/org/apache/nifi/cluster/coordination/flow/PopularVoteFlowElectionFactoryBean.java

```diff
@@ -44,8 +43,10 @@ public PopularVoteFlowElection getObject() throws Exception {
         }
 
         final Integer maxNodes = properties.getFlowElectionMaxCandidates();
-
-        final StringEncryptor encryptor = StringEncryptor.createEncryptor(properties);
```

Comment:
it seems like this building of the encryptor needs to only be in the framework/main code one time using the properties as shown here.  Then for all the tests we could have a util method.  Perhaps in nifi-mock?

Reply:
Will open a separate Jira to cover this and remove the deprecated method from the codebase as it is no longer used/exposed.